### PR TITLE
content: blog for 0.25.0

### PR DESCRIPTION
**Pull request by fntlnz**
Co-Authored-By: Leonardo Grasso <me@leonardograsso.com
Signed-off-by: Lorenzo Fontana <fontanalorenz@gmail.com>

**What type of PR is this?**

/kind content

**Any specific area of the project related to this PR?**

/area documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by fntlnz** [comment]
/hold

We want to wait for the release to be finished.

**Review by leogr — DISMISSED** [review]
LGTM

File: content/en/blog/falco-0-25-0.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+### Rules
+
+As always, our rules set is constantly improving and adapting to the constantly changing world.
+Many thanks to [@Kaizhe](https://github.com/Kaizhe),  [@nvanheuverzwijn](https://github.com/nvanheuverzwijn), [@admiral0](https://github.com/admiral0) and [@leogr](https://github.com/leogr). 
+
+
+
+## What's next?
+We just created the milestone for [0.26.0](https://github.com/falcosecurity/falco/milestone/12)  due by September 15, 2020
```
Comment (fntlnz) [inline content/en/blog/falco-0-25-0.md:57]:
```suggestion
We just created the milestone for [0.26.0](https://github.com/falcosecurity/falco/milestone/12)  due by September 15, 2020.
```

**Review by leogr — APPROVED** [review]
LGTM again :+1:

**Pull request merged**